lib/coophub/repos.ex

```diff
@@ -70,7 +73,8 @@ defmodule Coophub.Repos do
       orgs when is_map(orgs) ->
         orgs
         |> Map.values()
-        |> Enum.map(&Map.delete(&1, "repos"))
+        # Clear repos!
+        |> Enum.map(&Map.put(&1, :repos, []))
```

Comment:
I don't `delete` because the property should be present at the struct (although it supports to be deleted).